### PR TITLE
Port 2109 create a Timer docs

### PR DESCRIPTION
**Pull request by dvirsegev**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please extend the example inside the deep dive page, we need to show an example with a larger request body that has more content so the user understand that not only the timer property that expired is sent to him in the action event

File: docs/software-catalog/blueprint/timer-properties.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 6
+---
+
+# Timer Properties
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/timer-properties.md:5]:
```suggestion
# Timer properties
```

File: docs/software-catalog/blueprint/timer-properties.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        "title": "Timer property",
+        "type": "string",
+        "format": "Timer"
+    }
+}
+```
+
+## Timer properties deep dive
+
+Let's look at some examples of basic timer properties definitions to better understand how timer properties work:
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/timer-properties.md:27]:
```suggestion
Let's look at some examples of basic timer properties definitions to better understand how timer properties work.
```

File: docs/software-catalog/blueprint/timer-properties.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+In the following example, we create a timer property called locked, that will expire in 2 hours:
```
Comment (EllaFurman) [inline docs/software-catalog/blueprint/timer-properties.md:29]:
```suggestion
In the following example, we will create a timer property called locked, that will expire in 2 hours:
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**